test/testdata/infer/sealed_list_module.rb

```diff
@@ -154,8 +153,8 @@ def list_map_blk(xs, &blk)
 # This error is a bug. Just capturing the existing behavior.
 res2 = list_map_blk(xs) do |x|
   # This error is a bug. Just capturing the existing behavior.
-  T.reveal_type(x) # error: Revealed type: `<top>`
+  T.reveal_type(x) # error: Revealed type: `Integer`
   # This error is a bug. Just capturing the existing behavior.
-  x.even? # error: Method `even?` does not exist on `<top>`
+  x.even?
```

Comment:
I guess the "This error is a bug" comments here should be deleted?

Reply:
Thanks. I'm going to handle these in a follow-up PR so that I don't have to wait for another CI run to get this change in.